### PR TITLE
Mark cumulative

### PR DESCRIPTION
**Pull request by turbomam**

### CODE REVIEW

**Comment by cmungall** [comment]
shall we merge this?

**Comment by turbomam** [comment]
> shall we merge this?

Yeah. I'd like to make sure I haven't broken anything. I can work on that today and Monday.

- [ ] all command line scripts should remain active
- [ ] all tests should pass
- [ ] all GH actions should complete
- [ ] use (and automate) sensible doc generation and publication
- [ ] automate publication of package to PyPI

**Review by cmungall — APPROVED** [review]

File: .github/workflows/check-pull-request.yaml [hunk 1]
```diff
@@ -1,33 +1,76 @@
...
+      #----------------------------------------------      
+      - name: Install library
+        run: poetry install --no-interaction
+          
+      - name: Run image
+        uses: abatilo/actions-poetry@v2.0.0
+        # with:
+        #   poetry-version: ${{ matrix.poetry-version }}
+      - name: View poetry --help
+        run: poetry --help
```
Comment (sujaypatil96) [inline .github/workflows/check-pull-request.yaml:71]: Is there any specific reason for having to log available poetry commands in every run? I think we can remove this step, so we can reduce the amount of content being logged by the Action run.

Comment (turbomam) [reply .github/workflows/check-pull-request.yaml:71]: yeah, good catch. this was something I did while learning.

File: .github/workflows/check-pull-request.yaml [hunk 2]
```diff
@@ -1,33 +1,76 @@
...
+        run: poetry install --no-interaction --no-root
+        
+      #----------------------------------------------
+      #    install your root project, if required 
+      #----------------------------------------------      
+      - name: Install library
+        run: poetry install --no-interaction
+          
+      - name: Run image
+        uses: abatilo/actions-poetry@v2.0.0
```
[the remark this answers is not in the document]
Comment (turbomam) [reply .github/workflows/check-pull-request.yaml:67]: I thought you had to do this to install the poetry appliaction. If you know a better way, that's fine with me.

[the remark this answers is not in the document]
Comment (sujaypatil96) [reply .github/workflows/check-pull-request.yaml:67]: Ah, gotcha. So the poetry application is getting installed in this step: https://github.com/linkml/linkml-model-enrichment/blob/mark-cumulative/.github/workflows/check-pull-request.yaml#L28-L35

So I think it's okay to remove this step.

**Pull request merged**